github-org/action.yaml

```diff
@@ -10,10 +10,6 @@ inputs:
   token:
     description: GitHub Authentication Token
     required: true
-  args:
```

Comment:
Do we have to remove this?

Reply:
➕ to this. the `args` provides a method to pass any supported args, and create custom actions.

Reply:
Will have to figure something out how we can add this back.

Reply:
Ok, next iteration with another approach. No if, no wrapper, just not using the docker action.